site/content/docs/5.0/getting-started/parcel.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: docs
+title: Parcel
+description: Learn how to include Bootstrap in your project using Parcel.
+group: getting-started
+toc: true
+---
+
+## Install Parcel
+
+Install [Parcel Bundler](https://en.parceljs.org/getting_started.html).
+
+## Install Bootstrap
+
+[Install bootstrap]({{< docsref "/getting-started/download#npm" >}}) as a Node.js module using npm.
+
+Bootstrap depends on [Popper](https://popper.js.org/), which is specified in the `peerDependencies` property. This means that you will have to make sure to add both of them to your `package.json` using `npm install popper.js`.
+
+When all will be completed, your project will be structured like this:
+
+{{< highlight text >}}
+project-name/
+├── build/
+├── node_modules/
+│   └── bootstrap/
+│   └── popper.js/
+├── scss/
+│   └── custom.scss
+├── src/
+│   └── index.html
+│   └── index.js
+└── package.json
+{{< /highlight >}}
+
+## Importing JavaScript
+
+Import [Bootstrap's JavaScript]({{< docsref "/getting-started/javascript" >}}) in your app's entry point (usually `src/index.js`). You can import all our plugins in one file or separately if you require only a subset of them.
+
+{{< highlight js >}}
+// Import all plugins
+import * as bootstrap from 'bootstrap';
+
+// Or import only needed plugins
+import { Tooltip as Tooltip, Toast as Toast, Popover as Popover } from 'bootstrap';
+
+// Or import just one
+import Alert as Alert from '../node_modules/bootstrap/js/dist/alert';
+{{< /highlight >}}
+
+## Importing CSS
+
+To utilizie the full potential of Bootstrap and customize it to your needs, use the source files as a part of your project's bundling process.
+
+Create your own `scss/custom.scss` to [import Bootstrap's Sass files]({{< docsref "/customize/sass#importing" >}}) and then override the [built-in custom variables]({{< docsref "/customize/sass#variable-defaults" >}}).
+
+## Build app
+
+Include `src/index.js` before the closing `</body>` tag.
+
+{{< highlight html >}}
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+    <script src="./index.js"></script>
+  </body>
+</html>
+{{< /highlight >}}
+
+### Edit `package.json`
+
+Add dev and build scripts in your `package.json` file.
+
+{{< highlight json >}}
+"scripts": {
+  "dev": "parcel ./src/index.html",
+  "prebuild": "rm -rf build",
```

Comment:
I don't really like `rm -rf` because it doesn't work on Windows.

Reply:
See there https://github.com/parcel-bundler/parcel/issues/1234

Reply:
Yes, I'm aware of `shx`, but you don't use it here nor there is a comment that this won't work on vanilla Windows cmd :)